server/routes/request.ts

```diff
@@ -282,9 +319,23 @@ requestRoutes.get('/count', async (_req, res, next) => {
     const pendingCount = await requestRepository.count({
       status: MediaRequestStatus.PENDING,
     });
-    const approvedCount = await requestRepository.count({
-      status: MediaRequestStatus.APPROVED,
-    });
+
+    const approvedCount = await requestRepository
+      .createQueryBuilder('request')
+      .leftJoinAndSelect('request.media', 'media')
+      .leftJoinAndSelect('request.seasons', 'seasons')
+      .leftJoinAndSelect('request.modifiedBy', 'modifiedBy')
+      .leftJoinAndSelect('request.requestedBy', 'requestedBy')
+      .where('request.status = :requestStatus', {
+        requestStatus: MediaRequestStatus.APPROVED,
+      })
+      .andWhere(
+        '(request.is4k = false AND media.status != :availableStatus) OR (request.is4k = true AND media.status4k != :availableStatus)',
+        {
+          availableStatus: MediaStatus.AVAILABLE,
+        }
+      )
```

Comment:
Why does the approved count filter out available titles? Just because they become available doesn't make them not approved?

Reply:
Yes, but the point here was to separate available requests from approved, still-processing requests, if you take a look at #656.  Do we want to rename the filter option to something else?

Reply:
This is the count endpoint and it says "approved count" so it should be the number of approved requests, regardless of their availability. It doesn't say "available count"?

Reply:
We shouldn't change it either. If you want to add a new count for available items, sure? But I don't see why we would change the behavior for "approved".

Reply:
I think that an "approved" count that includes available requests isn't particularly useful, however.  Wouldn't we want to know how many requests are approved and still pending?  That's actually what the Ombi endpoint returns and it was the original intention of this endpoint.  It's far more useful this way, since the "approved" count will otherwise just be a counter that keeps growing/increasing.

Reply:
Then we should add another field like "processing" because approved to me means returning the count of approved items.

Reply:
Sure!  That sounds like a good idea.

Reply:
I'll make the change to both endpoints and add the additional filter option to the request list.